src/components/Articel.jsx

```diff
@@ -1,39 +1,42 @@
 import { useEffect, useState } from "react";
-import { selectArticel,selectComments } from "../api";
-import {ArticelCard} from './Articel-card'
+import { selectArticel, selectComments } from "../api";
+import { ArticelCard } from "./Articel-card";
 import { useParams } from "react-router-dom";
 import NavBar from "./NavBar";
-import Commentbox from './comment-box'
+import Commentbox from "./comment-box";
 export default function Articel() {
-  let {id,topic} = useParams();
+  let { id, topic } = useParams();
   const [articel, setArticel] = useState([]);
-  const [comments,setComments] = useState([]);
+  const [comments, setComments] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   useEffect(() => {
     setIsLoading(true);
-    selectArticel(id).then((article) => { 
-      console.log(article)
+    selectArticel(id)
+      .then((article) => {
+        console.log(article);
         setArticel(article);
-      
-      return selectComments(id)
-    }).then((comments)=>{
-      setComments(comments);
-      setIsLoading(false);
-    })
-  }, [topic,id]);
 
-  if (isLoading) return <p>loading..</p>;
+        return selectComments(id);
+      })
+      .then((comments) => {
+        setComments(comments);
+        setIsLoading(false);
+      });
+  }, [topic, id]);
```

Comment:
why is topic in here? If you're rendering a single article it's based on it's ID not topic